Makefile

```diff
@@ -16,7 +17,7 @@ default: dev
 quickdev: generate
 	@CGO_ENABLED=0 go build -i -tags='$(BUILD_TAGS)' -o bin/${TOOL}
 dev: generate
-	@CGO_ENABLED=0 BUILD_TAGS='$(BUILD_TAGS)' VAULT_DEV_BUILD=1 sh -c "'$(CURDIR)/scripts/build.sh'"
+	@CGO_ENABLED=0 BUILD_TAGS='$(BUILD_TAGS)' VERSION='$(VERSION)' VAULT_DEV_BUILD=1 sh -c "'$(CURDIR)/scripts/build.sh'"
```

Comment:
Should the version tag be added to the `bin` target as well?

Reply:
Good point, I'll add it.